loguru.hpp

```diff
@@ -2301,6 +2319,30 @@ namespace loguru
 
 	// ------------------------------------------------------------------------
 
+	static void print_preamble_header(char* out_buff, size_t out_buff_size)
+	{
+		long pos = 0;
+		snprintf(out_buff, out_buff_size, ""); // Make sure there is a '\0' and handle out_buff_size==0
+ 		if(g_preamble_date) {
+ 			pos += snprintf(out_buff + pos, out_buff_size - pos, "date       ");
+ 		}
+ 		if(g_preamble_time) {
+ 			pos += snprintf(out_buff + pos, out_buff_size - pos, "time         ");
+ 		}
+ 		if(g_preamble_uptime) {
+ 			pos += snprintf(out_buff + pos, out_buff_size - pos, "( uptime  ) ");
+ 		}
+ 		if(g_preamble_thread) {
+ 			pos += snprintf(out_buff + pos, out_buff_size - pos, "[%-*s]", LOGURU_THREADNAME_WIDTH, " thread name/id");
+ 		}
+ 		if(g_preamble_file) {
+ 			pos += snprintf(out_buff + pos, out_buff_size - pos, "%*s:line     ", LOGURU_FILENAME_WIDTH, "file");
+ 		}
+ 		if(g_preamble_verbose) {
+ 			pos += snprintf(out_buff + pos, out_buff_size - pos, "v| ");
+ 		}
```

Comment:
These new lines have a space before the first tab:

<img width="694" alt="screen shot 2018-01-21 at 21 03 05" src="https://user-images.githubusercontent.com/1148717/35198394-8b7c7f2e-feee-11e7-80a9-328fa84e6bf0.png">